contrib/macos/create-app-bundle.sh

```diff
@@ -35,6 +35,16 @@ install -m 644 "README"                                    "$dst/SharedSupport/m
 install -m 644 "docs/README.video"                         "$dst/SharedSupport/video.txt"
 # install -d     "$dst/Contents/_CodeSignature/" # ?
 
+install -m 755 "contrib/macos/Start DOSBox.command"        "dist/"
+
+# Hide extension of "Start DOSBox.command" in Finder
+xattr -x -w com.apple.FinderInfo "00 00 00 00 00 00 00 00 00 10 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00" "dist/Start DOSBox.command"
+
+# Set up visual appearance of the root folder of the DMG image
+install -d "dist/.hidden"
+install -m 644 "contrib/macos/background/background.tiff"  "dist/.hidden/"
+install -m 644 "contrib/macos/DS_Store"                    "dist/.DS_Store"
+
```

Comment:
The `dist/` area is part of the `$dst` top-level variable; so we should adjust the top-level variables so they both are derived from the same truth or data.  Otherwise, there are multiple sources of "truth":

![2022-12-15_09-39](https://user-images.githubusercontent.com/1557255/207929982-37041db2-e03c-4423-849a-8f72c51e314f.png)

Maybe something like:
```shell
app_name="DOSBox Staging"
dist_dir="dist"
content_dir="$dist_dir/$app_name.app/Contents"
```

(we could re-use `$app_name` and $dist_dir` in the `hdiutil create` call too, on line 46).

Reply:
I kinda considered that, actually... but then didn't do it. Making the change.